CHANGELOG.md

```diff
@@ -73,6 +73,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
   in `allowed net users` / `allowed net groups`, and apptainer is installed with
   setuid privileges. Not currently supported with `--fakeroot`.
 - Go version 1.22 is now required.
+- Refactor image arch variation using go-containerregistry's platform.
```

Comment:
Actually now that I see the proposed CHANGELOG entry I see that it doesn't affect user experience at all so it probably should not be there.   I'll go ahead and merge it as-is though and plan on moving it to a separate "internal changes" section of the release announcement when I prepare the release and change it to describe the general internal refactoring to use go-containerregistry.